crates/hyperdrive-math/src/short/max.rs

```diff
@@ -569,7 +583,8 @@ mod tests {
             let max_iterations = 7;
             let open_vault_share_price = rng.gen_range(fixed!(0)..=state.vault_share_price());
             // We need to catch panics because of overflows.
-            let actual = panic::catch_unwind(|| {
+            // TODO: This should just call calculate_absolute_max_short since that's the thing we're testing against.
```

Comment:
is this for later or now?

Reply:
I'll do this in a follow-up PR that includes a bunch of test fixes.